controllers/cloudflare_api.go

```diff
@@ -2,69 +2,45 @@ package controllers
 
 import (
 	"bytes"
+	"context"
 	"crypto/rand"
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
 	"net/http"
 	"net/url"
 
+	"github.com/cloudflare/cloudflare-go"
 	"github.com/go-logr/logr"
 )
 
-// CLOUDFLARE_ENDPOINT is the Cloudflare API base URL from https://api.cloudflare.com/#getting-started-endpoints.
-const CLOUDFLARE_ENDPOINT = "https://api.cloudflare.com/client/v4/"
-
 // TXT_PREFIX is the prefix added to TXT records for whom the corresponding DNS records are managed by the operator.
 const TXT_PREFIX = "_managed."
 
```

Comment:
I do not think this is needed anymore since the TXT is being added on the CNAME itself, right? 
```suggestion
```

Reply:
I agree that upon inspection, its not used anymore. I'm not sure the TXT record will have that prefix without this though, it might just be the same as `fqdn`. Probably need to test this, but  can remove it for now.

Reply:
I remember now, you cannot have any other record types with CNAME.

So, we do need that prefix, and need to add it back in the getManagedTxt and delete functions, sorry

https://serverfault.com/questions/834320/adding-both-cname-and-txt-dns-records-for-one-subdomain

Reply:
Fixed in https://github.com/adyanth/cloudflare-operator/pull/77/commits/062057541296aa766940186a2851b750e20c7602